### DISCUSSION TITLE
Can we adjust pixel shape/roundness?

### DISCUSSION THREAD

**Discussion by RDreamerGr** [Shane32/QRCoder#298] [Q&A]
Hi, is it possible to adjust the pixel roundness to make the qr codes more modern looking/variated?

Also is it possible to have a gradient color scheme?

**Comment by codebude** [comment]
Currently (08/2021) this isn't supported by the lib, but you can compile QRCoder yourself and add a 3rd party QRCode renderer class (https://github.com/AlexandreZaytsev/QRCoderArt/blob/master/ArtQRCode.cs), thus you should reach your aim.
